### PR TITLE
Ss patch1

### PR DESCRIPTION
**Pull request by stevens34400**

### CODE REVIEW

File: app/src/main/java/adapters/BaseCommentAdapter.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    @Override
+    public void onBindViewHolder(@NonNull BaseCommentAdapter.ViewHolder holder, int position) {
+        Comment comment = comments.get(position);
+        holder.bind(comment);
+    }
+
+    @Override
+    public int getItemCount() {
+        return 0;
```
Comment (akp5896) [inline app/src/main/java/adapters/BaseCommentAdapter.java:44]: comments.size()

File: app/src/main/java/com/example/sc2infoapp/TeamActivity.java [hunk 2]
```diff
@@ -76,7 +77,8 @@ protected void onCreate(Bundle savedInstanceState) {
 
 
 
-        matches = new ArrayList<>();;
+        matches = new ArrayList<>();
+        externalMatches = new ArrayList<>();
```
Comment (akp5896) [inline app/src/main/java/com/example/sc2infoapp/TeamActivity.java:81]: ???

Comment (stevens34400) [reply app/src/main/java/com/example/sc2infoapp/TeamActivity.java:81]: Didn't touch TeamActivity, might've been from the merge I got from you last night

Comment (akp5896) [reply app/src/main/java/com/example/sc2infoapp/TeamActivity.java:81]: Hm, by github history it seems that it's from base commetn commit

File: app/src/main/java/adapters/BaseCommentAdapter.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    }
+
+    public class ViewHolder extends RecyclerView.ViewHolder {
+        TextView tvUser;
+        TextView tvComment;
+
+        public ViewHolder(@NonNull View itemView) {
+            super(itemView);
```
Comment (akp5896) [inline app/src/main/java/adapters/BaseCommentAdapter.java:52]: You also need author's username

**Review by justin212553 — APPROVED** [review]

**Review by akp5896 — APPROVED** [review]

**Pull request merged**